inc/admin/dashboard/main.php

```diff
@@ -232,6 +234,13 @@ private function get_localization() {
 			'tpcAdminURL'         => admin_url( 'themes.php?page=tiob-starter-sites' ),
 		];
 
+		if ( ! $this->is_valid_license() ) {
+			$starter_sites_desc                             = $data['strings']['starterSitesCardDescription'];
+			$neve_pro_upsell                                = esc_html__( 'Upgrade to the Pro version and get instant access to all Premium Starter Sites; including Expert Sites, and much more.', 'neve' );
+			$starter_sites_desc_upsell                      = $starter_sites_desc . '<br/><br/>' . $neve_pro_upsell;
+			$data['strings']['starterSitesCardDescription'] = $starter_sites_desc_upsell;
+		}
+
```

Comment:
Let's not do this here, we can add it directly as a child in the specific `Card` in JS as another paragraph with the `description` class.